imports/blocks/give/fieldsets/confirm/index.js

```diff
@@ -71,22 +76,25 @@ export default class Confirm extends Component {
     );
   }
 
-  choose = (e) => {
+  choose = (e: Event) => {
     e.preventDefault();
 
-    const { id } = e.currentTarget;
-    let act = {};
-    for (const account of this.props.savedAccounts) {
-      if (Number(account.id) === Number(id)) {
-        act = account;
-        break;
+    const target = e.currentTarget;
+    if (target instanceof HTMLInputElement) {
+      const { id } = target;
+      let act = {};
+      for (const account of this.props.savedAccounts) {
+        if (Number(account.id) === Number(id)) {
```

Comment:
I'm not sure these will always be castable to Numbers. As we switch to node-ids everywhere, this will probably be two arbitrary strings. Can we change to string casting comparison?